airflow/www/static/js/dag/details/Header.tsx

```diff
@@ -105,7 +106,7 @@ const Header = () => {
           onClick={clearSelection}
           _hover={isDagDetails ? { cursor: "default" } : undefined}
         >
-          <BreadcrumbText label="DAG" value={dagId} />
+          <BreadcrumbText label="DAG" value={dagDisplayName} />
```

Comment:
Should we use dagId as a backup or will displayName always exist?

Reply:
Thanks! Didi not notice - or was this a late change due to your rework the last days? Anyway, thanks... fixed!

Reply:
And... yes the model always makes a fallback to task ID/ DAG id if display name not present.